data/gaTable.yaml

```diff
@@ -983,3 +983,23 @@
   projectContactEmail: 'mpavlick@usgs.gov'
   analyticsContact: 'Meredith Pavlick'
   analyticsContactEmail: 'mpavlick@usgs.gov'
+
+- longName: "VizLab: Hurricane Harvey's Water Footprint"
+  shortName: VizLabHarvey
+  accountName: Vizlab
+  viewID: '159018840'
+  accountId: '78530187'
+  webPropertyId: UA-78530187-7
+  internalWebPropertyId: '157530047'
+  viewName: All Web Site Data
+  timezone: America/Chicago
+  websiteUrl: https://owi.usgs.gov/vizlab/hurricane-harvey
+  botFilteringEnabled: ''
+  description: >-
+    The VizLab Hurricane Harvey Water Footprint (VizLabHarvey) application demonstrates
+    Hurricane Harvey's approach to the U.S. Gulf coast on August 25-31, 2017. This
+    visualization shows the hurricane's impact on precipitation and streamflow.
+  projectContact: 'Jordan Walker'
```

Comment:
sadness, but maybe we should phase jordan out now rather than trying to remember in 1 month?

Reply:
Good question, posted in slack

Reply:
i saw that. it's OK with me if we proceed with Jordan listed - he can stay in charge until 9/30 =)